image.yaml

```diff
@@ -1,7 +1,7 @@
 schema_version: 1
 
 name: "jboss-processserver-6/processserver64-openshift"
-version: "1.2"
+version: "1.3"
 description: "Platform for executing business rules on JBoss BPMS Intelligent Process Server 6.4."
 from: "jboss-kieserver-6/kieserver64-openshift:1.2"
```

Comment:
Shouldn't it be 1.3?

Reply:
yes probably. this is a good point, must be careful with this family of images.